storage/innobase/include/trx0trx.h

```diff
@@ -734,14 +734,27 @@ struct trx_t {
 	Transitions to COMMITTED are protected by trx_t::mutex. */
 	trx_state_t	state;
 #ifdef WITH_WSREP
-	/** whether wsrep_on(mysql_thd) held at the start of transaction */
-	bool		wsrep;
+	/** (1) whether wsrep_on(mysql_thd) held at the start of transaction
+	    (2) whether wsrep_thd_is_BF(mysql_thd) held at start of transaction
+	    (4) whether BF thread is performing unique secondary index scan
+	**/
+	static const byte TRX_WSREP_ON = 1;
+	static const byte TRX_WSREP_BF = 2;
+	static const byte TRX_WSREP_UK_SCAN = 4;
+	byte		wsrep;
```

Comment:
There is no Doxygen comment for the data member `wsrep` anymore. The constants are missing `constexpr`.

Reply:
I will fix the Doxygen (not sure its really used) but I used here same const as undo_no_t constant (see 10.6 version where constexpr was used). Can you use C++11 on 10.4?

Reply:
d9613b750c6d975beed0e1d7f0cb1a8dd3ad106b enabled the use of C++11